sources/packages/web/src/components/common/OfferingForm.vue

```diff
@@ -66,14 +70,13 @@ export default defineComponent({
     cancel: null,
   },
   props: {
-    data: {
-      type: Object as PropType<OfferingFormModel>,
+    offeringId: {
```

Comment:
wasn't the plan to use both `id` and `data` as optional?

Reply:
That was the plan @ann-aot. If after the refactor there is no need to have the data we can leave it with only the offeringId assuming that Ministry and Institution would be able to retrieve the offering data nicely.
Again, we can remove `data` only if the refactor ensures that it is no longer needed.

Reply:
@andrewsignori-aot if that is the case, institutions need to use locationId to get the offering right? as we validate hasLocationAccess

Reply:
Fixed it.